CMakeLists.txt

```diff
@@ -54,6 +54,12 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
     set(IS_DEBUG_BUILD ON)
 endif()
 
+option(HIDAPI_ENABLE_ASAN "Build HIDAPI with ASAN address sanitizer instrumentation" ON)
+
+if(HIDAPI_ENABLE_ASAN)
+    add_compile_options(-fsanitize=address)
```

Comment:
maybe:
```
set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address")
```
would work better.
(I'm refering to failed non-MSVC builds).
At least that's how I used it in my other projects.